app/graphql/types/subscription_type.rb

```diff
@@ -14,4 +14,8 @@ def weather_published(latitude:, longitude:); end
   field :announcement_published, Types::AnnouncementType, description: "An announcement was published"
 
   def announcement_published; end
+
+  field :schema_sha, String, description: "Updated SHA value"
+
+  def schema_sha; end
```

Comment:
there is something happening I don't understand. How is this subscription field connected to the poll worker?